stable/airflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 0.10.0
```

Comment:
Sadly this is a impacting change, I recommend to jump to 0.11.0

Reply:
thanks, I have bumped the version

Reply:
Great :)